coding-the-legislation/35_periods.md

```diff
@@ -1,25 +1,95 @@
 # Periods
 
+A period can be a month, a year, `n` successive months or `n` successive years.
+
+
+## Periods for variable
+
 Most of the quantities calculated in openfisca can change over time. Therefore, each formula calculates a variable for a person (or a family, etc.) **for a given period**.
 
 This period is always the second argument of the formulas :
+
 ```py
+class var(Variable):
+    column = FloatCol
+    entity = Person
+    label = u"some variable"
```

Comment:
Give a better example name.

It should at least have proper casing, and demonstrate the kind of precision we expect from contributors.

The best documentation has consistent examples exploring a clearly scoped ecosystem, possibly imaginary but still consistent.

For example, you might use examples that always relate to how much pocket money a child gets from her parents, or how a super-serious pizzeria would price its pizzas according to their ingredients and how long they've been in stock. This makes the documentation funnier to read, but also clearer to the newbie, since you develop a consistent, small-scale world.